.github/ISSUE_TEMPLATE/10-action copy.yml

```diff
@@ -0,0 +1,62 @@
+name: Local Meetup Tracking/Action Item
+description: Use this issue type to track and manage local meetup organization tasks. Avoid using this issue type for tasks expected to be closed within hours.
+title: "[<Tracking>/<Action>] Cloud Native Sustainability Week 2024 <Meetup Location>"
+labels: ["issue/needs-triage", "board/unassigned"]
+projects: ["cncf/10"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing to the TAG!
+        Please remember that an issue is not the place to ask questions.
+        For inquiries, please refer to the README on how to reach us: [Contact](https://github.com/cncf/tag-env-sustainability#contact).
+        Thank you! :)
+  - id: description
+    type: textarea
+    attributes:
+      label: Description
+      description: |
+        Please provide the following details:
+        * **Location/Region:**
+        * **Organizers:**
+        * **Sponsors:**
+        * **Date:**
+        * **Event link:**
+    validations:
+      required: true
+  - id: details
+    type: textarea
+    attributes:
+      label: Details
+      description: What are the goals and activities planned for the meetup?
+    validations:
+      required: true
+  - id: todo
+    type: textarea
+    attributes:
+      label: To-Do
+      description: What specific actions are needed to organize this meetup?
```

Comment:
We could add potentially

- Joined one of the TAG ENV Sustainability Week Sync to talk about your event
- Found a sponsor for the event
- Found a location for the event
- Set a date for the event
- Announced the event
- Reached out to the Cloud Native Sustainability Week Organizer team to promote the event